cmd/jb-server/main.go

```diff
@@ -89,6 +89,11 @@ func main() {
 			EnvVar: "JUPITER_BRAIN_DATABASE_POOL_SIZE,DATABASE_POOL_SIZE",
 			Value:  2,
 		},
+		cli.StringFlag{
+			Name:   "database-app-name",
+			Usage:  "app name",
```

Comment:
Could we make this a little more descriptive? Maybe something like "application name for Postgres connections"?